controllers/argocd/util.go

```diff
@@ -669,7 +641,7 @@ func (r *ReconcileArgoCD) redisShouldUseTLS(cr *argoproj.ArgoCD) bool {
 	tlsSecretName := types.NamespacedName{Namespace: cr.Namespace, Name: common.ArgoCDRedisServerTLSSecretName}
 	err := r.Client.Get(context.TODO(), tlsSecretName, &tlsSecretObj)
 	if err != nil {
-		if !apierrors.IsNotFound(err) {
```

Comment:
I would prefer we keep this as `apierrors` as it is more descriptive and allows us to separately import the go errors pkg as well, if needed

Reply:
updated